src/main/java/net/minecraftforge/event/world/PistonEvent.java

```diff
@@ -37,7 +37,6 @@
     private final PistonMoveType moveType;
 
     /**
-     * @param world
```

Comment:
Again not sure how I feel about removing doc for only a single parameter. I think if we document the parameters than we should have some small description for all of them.

Reply:
Okay we can remove this one. But I find it odd that it is called world while being Level type. Is this a leftover from the mojmap transition? If so I think we could go around fixing these as well in a different PR.

Reply:
Yep. I'll create another PR that aims to replace references to MCP names in parameters/Javadoc soon.